grid/p5js/grid-closure/index.js

```diff
@@ -0,0 +1,16 @@
+function setup() {
+  createCanvas(500, 500);
```

Comment:
Please assign it to a parent element as statet in my other review for color-contrast